addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -345,6 +345,8 @@ export class HtmlField extends Component {
             const t = document.createElement('T');
             t.setAttribute('t-out', dynamicPlaceholder);
             this.wysiwyg.odooEditor.execCommand('insert', t);
+            // Ensure the dynamic placeholder <t> element is sanitized.
+            this.wysiwyg.odooEditor.sanitize(t.parentElement);
```

Comment:
The process that ensure the T element are inlined,

> odooEditor.js, line 748 >  `this._pluginCall('sanitizeElement', [target]);`

expect to receive the t element parent.
> QWebPlugin.js, line 36 > `sanitizeElement(subRoot)`

I can remove the `.parentElement` from html_field.js, and add it back in odooEditor.js sanitize method, if you prefer.

Reply:
My opinion is that whoever needs to sanitize a given node should not have to read the code in another file to understand that they must give _another_ node (here the parent) as the node to sanitize instead of the actual node they want to sanitize. Therefore, I think the call should be passed `t` and not `t.parentElement`. I think it is the responsibility of the sanitization code to handle the `t` case appropriately and sanitize other nodes if needed. Something similar is already done for lists IIRC.